server/database.js

```diff
@@ -209,14 +209,9 @@ class Database {
         let config = {};
 
         let mariadbPoolConfig = {
-            pool: {
```

Comment:
I added it wrongly, that might be the reason why `min: 0` was not working. Will test it again.